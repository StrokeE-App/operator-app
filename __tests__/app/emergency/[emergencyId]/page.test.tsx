import {render} from '@testing-library/react';
import EmergencyPage from '@/app/emergency/[emergencyId]/page';

describe('EmergencyPage', () => {
	it('matches snapshot', () => {
		const {container} = render(<EmergencyPage params={Promise.resolve({emergencyId: '123'})} />);
		expect(container).toMatchSnapshot();
	});
});
