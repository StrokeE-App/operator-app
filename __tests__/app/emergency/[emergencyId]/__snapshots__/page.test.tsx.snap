// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmergencyPage matches snapshot 1`] = `<div />`;
